src/Microsoft.TestPlatform.PlatformAbstractions/netstandard/Tracing/PlatformEqtTrace.cs

```diff
@@ -33,38 +33,32 @@ public class PlatformEqtTrace : IPlatformEqtTrace
 
     public void WriteLine(PlatformTraceLevel traceLevel, string message)
     {
-        if (!ShouldTrace(traceLevel))
-        {
-            return;
-        }
-
         var level = Enum.GetName(typeof(PlatformTraceLevel), traceLevel);
 
+# if NETSTANDARD1_3
+        System.IO.File.AppendAllText("C:\\temp\\log.txt", $"[{level}] {message}");
```

Comment:
I think we should be using env var for temp folder.

Reply:
this is just some mixup of changes I needed for winui work. I probably added them here and built the dll so I can copy paste it into my debug package, and forgot to revert, because I thought I am in a different repo. Will revert.

Reply:
reverted